Jamulus.pro

```diff
@@ -95,20 +95,28 @@ win32 {
     contains(CONFIG, "jackonwindows") {
         message(Using Jack instead of ASIO.)
 
-        !exists("C:/Program Files (x86)/Jack/includes/jack/jack.h") {
-            message(Warning: jack.h was not found at the usual place, maybe jack is not installed)
-        }
+        contains(QT_ARCH, "x86") {
+        	ProgramFilesDir = read_registry(HKLM, ProgramFilesDir[, 32])
+        	libjack-name = libjack.lib
+        } else {
+            ProgramFilesDir = read_registry(HKLM, ProgramFilesDir[, 64])
+		    libjack-name = libjack64.lib	
+          }
+          !exists("$${ProgramFilesDir}/JACK2/include/jack/jack.h") {
+            message("Warning: jack.h was not found at the usual place, maybe JACK2 is not installed")
+          }
+
```

Comment:
Indentation seems to be off here and further below.

Reply:
ok in
5aa0168